src/LoggerListeners.cpp

```diff
@@ -280,7 +282,9 @@ cFileListener::cFileListener(void)
 {
 	cFile::CreateFolder(FILE_IO_PREFIX + AString("logs"));
 	AString FileName;
-	FileName = Printf("%s%sLOG_%d.txt", FILE_IO_PREFIX, "logs/", (int)time(nullptr));
+	auto time = std::chrono::system_clock::now();
+	FileName = Printf("%s%sLOG_%d.txt", FILE_IO_PREFIX, "logs/",
+			std::chrono::duration_cast<std::chrono::duration<int, std::milli>>(time.time_since_epoch()).count());
```

Comment:
If it weren't, Lua would fail on us, too, it exports the `time(NULL)` as `os.time()` API